.github/workflows/merge_queue.yml

```
@@ -0,0 +1,785 @@
+            -ref "${{ needs.context.outputs.head_sha }}"
+            -owner "${{ needs.context.outputs.base_repo_owner }}"
+            -repo "${{ needs.context.outputs.base_repo_name }}"
+            -state error
+            -context "test/cleanup"
+            -description "Destroying test fixtures cancelled"
+            -url "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+  results:
+    runs-on: ${{ fromJSON(inputs.runs-on) }}
```

actionlint (1.7.4)

property "runs-on" is not defined in object type {}

(expression)